test/e2e/toolchaincluster_test.go

```diff
@@ -174,3 +176,86 @@ func caBundle(bundle string) clusterOption {
+	t.Cleanup(func() {
+		err := hostAwait.ScaleDeployment(hostAwait.Namespace, "host-operator", 1)
+		require.NoError(t, err)
+	})
+	// before creating a batch of users, let's remove all remainings of previous tests, so
+	// we know exactly what numbers we're dealing with
+	userSignups := &toolchainv1alpha1.UserSignupList{}
+	err := hostAwait.Client.List(context.TODO(), userSignups, client.InNamespace(hostAwait.Namespace))
+	require.NoError(t, err)
+	for _, userSignup := range userSignups.Items {
+		err := hostAwait.Client.Delete(context.TODO(), &userSignup)
+		require.NoError(t, err)
+	}
+	// then wait until all UserSignups have been deleted
+	for _, userSignup := range userSignups.Items {
+		hostAwait.WaitUntilUserSignupDeleted(userSignup.Name)
+	}
+	// now we can create our new users ;)
+	CreateMultipleSignups(t, ctx, hostAwait, memberAwait, 5) // 5 external users
+
+	// when
+	metricsAssertion := InitMetricsAssertion(t, hostAwait, []string{memberAwait.ClusterName, member2Await.ClusterName})
+
+	t.Run("verify metrics are still correct after restarting pod", func(t *testing.T) {
+		// when
+		err := hostAwait.DeletePods(client.MatchingLabels{"name": "host-operator"})
+		// then
+		require.NoError(t, err)
+		metricsAssertion.WaitForMetricDelta(MasterUserRecordsPerDomainMetric, 0, "domain", "external")                       // unchanged compared to before deleting the pod
+		metricsAssertion.WaitForMetricDelta(UsersPerActivationsAndDomainMetric, 0, "activations", "1", "domain", "external") // unchanged compared to before deleting the pod
+	})
+
+	t.Run("tampering ToolchainStatus", func(t *testing.T) {
+
+		t.Run("verify metrics are still correct after restarting pod but not forcing recount", func(t *testing.T) {
+			// given
+			hostAwait.UpdateToolchainConfig(testconfig.Metrics().ForceSynchronization(false))
+			err := hostAwait.ScaleDeployment(hostAwait.Namespace, "host-operator", 0) // make sure it's "shut-down" while tampering the ToolchainStatus resource
+			require.NoError(t, err)
+			// tamper the values in ToolchainStatus.status.metrics
+			toolchainStatus, err := hostAwait.WaitForToolchainStatus()
+			require.NoError(t, err)
+			toolchainStatus.Status.Metrics[toolchainv1alpha1.MasterUserRecordsPerDomainMetricKey] = toolchainv1alpha1.Metric{
+				"external": toolchainStatus.Status.Metrics[toolchainv1alpha1.MasterUserRecordsPerDomainMetricKey]["external"] + 1, // increase current value by 1
+			}
+			toolchainStatus.Status.Metrics[toolchainv1alpha1.UserSignupsPerActivationAndDomainMetricKey] = toolchainv1alpha1.Metric{
+				"1,external": toolchainStatus.Status.Metrics[toolchainv1alpha1.UserSignupsPerActivationAndDomainMetricKey]["1,external"] + 1, // increase current value by 1
+			}
+			err = hostAwait.Client.Status().Update(context.TODO(), toolchainStatus)
+			require.NoError(t, err)
+			// when restarting the pod
+			err = hostAwait.ScaleDeployment(hostAwait.Namespace, "host-operator", 1)
+			require.NoError(t, err)
```

Comment:
I'm just curious - is it really safe to do that? Won't be the deployment scaled up by the operator controller in the meantime?

Reply:
no, the host-operator pod get deleted, that's all.